src/utils/locationInfo.js

```diff
@@ -56,4 +56,16 @@ const getStreetAddress = async (lat, lng) => {
   return results[0].formatted_address
 }
 
-export { getFormattedLocationInfo, getLocationInfo, getStreetAddress }
+/**
+ * Determines if seasonality information should be rendered
```

Comment:
Maybe make this comment more descriptive of the function rather than its use context outside this file.